app/views/requests/_request_type.html.erb

```diff
@@ -0,0 +1,8 @@
+<% case request_type %>
```

Comment:
This seems like overkill for a partial (which comes with some slowdown). It's just a `<div>` with some text which we can call `titleize` on.